contracts/Timelock.sol

```diff
@@ -45,7 +45,7 @@ contract Timelock {
     mapping(address => bool) public pendingAdminAccepted;
     mapping(bytes32 => bool) public queuedTransactions;
 
-    constructor(address _admin, uint256 _delay) public {
+    constructor(address _admin, uint256 _delay) {
```

Comment:
Why did we remove the `public` keyword here?

Reply:
Constructor doesn't need that keyword, it throws a warning